src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -3482,7 +3482,7 @@ def create_lb_frontend_ip_configuration(
 
 def update_lb_frontend_ip_configuration_setter(cmd, resource_group_name, load_balancer_name, parameters, gateway_lb):
     aux_subscriptions = []
-    if gateway_lb is not None:
+    if is_valid_resource_id(gateway_lb):
```

Comment:
Could you explain why the `gateway_lb` will be deleted if the subscription for `gateway_lb` is added to `aux_subscription`?